python_modules/libraries/lakehouse/lakehouse/house.py

```diff
@@ -135,6 +288,10 @@ def get_computed_asset_solid_def(self, computed_asset, assets_in_pipeline):
                 )
                 input_defs.append(input_def)
 
+        # Add a `Nothing` input if requested and if this asset has no input definitions.
```

Comment:
It's possible that even if we do have input_defs we might want to depend on some solids, right?

Reply:
I could be wrong but I thought the only assets that would have input_defs are those which are computed from other assets (i.e. they _aren't_ source assets). The idea here is to only add `Nothing` inputs to solids corresponding to source assets; I can't think why we'd need to explicitly add the `Nothing` input for a computed asset, since it would already have a transient dependency.

I admit I'm a bit iffy on the details though, so sorry if that's unclear (or wrong)!